src/Data/List/Relation/Binary/Sublist/Propositional/Properties.agda

```diff
@@ -47,3 +48,20 @@ module _ {p} {P : Pred A p} where
     cong here ∘′ subst-injective x≡y ∘′ here-injective
   lookup-injective {p = _ ∷ _}    {there v} {there w} =
     cong there ∘′ lookup-injective ∘′ there-injective
+
+------------------------------------------------------------------------
+-- Relationships to other predicates
+
+module _ {a} {A : Set a} where
+  open import Relation.Binary using (_Respects_)
+
+  All-resp-⊆ : ∀ {ℓ} {P : Pred A ℓ} → (All P) Respects (flip _⊆_)
+  All-resp-⊆ []          []       = []
+  All-resp-⊆ (_    ∷ʳ p) (_ ∷ xs) = All-resp-⊆ p xs
+  All-resp-⊆ (refl ∷  p) (x ∷ xs) = x ∷ All-resp-⊆ p xs
+
+  Any-resp-⊆ : ∀ {ℓ} {P : Pred A ℓ} → (Any P) Respects _⊆_
+  Any-resp-⊆ [] ()
```

Comment:
Is this lookup? https://agda.github.io/agda-stdlib/Data.List.Relation.Binary.Sublist.Heterogeneous.html#1916

Reply:
derp 😆

Reply:
You think it makes sense to re-export `lookup` as `Any-resp-⊆` to help someone seeking it by that name?

Reply:
I get confused myself but AFAIU `Any-resp-⊆` would be a special case of
`lookup` rater than a mere re-export? Which is fair enough in terms of
name / user expectations.

Reply:
I'm re-exporting `Data.List.Relation.Binary.Sublist.Propositional.lookup` rather than `Data.List.Relation.Binary.Sublist.Heterogeneous.lookup`; i believe the former has the correct type.